docker/server/Dockerfile

```diff
@@ -4,6 +4,8 @@ RUN apt-get update && apt-get install --yes --no-install-recommends postgresql-c
 
 RUN pip install --upgrade pip && pip install poetry
 
+RUN pip install sendgrid
+
```

Comment:
how come we are install sendgrid through docker? We are using poetry as our package manager - so if its installed with poetry it will automatically install all dependencies with the line above

Reply:
> how come we are install sendgrid through docker? We are using poetry as our package manager - so if its installed with poetry it will automatically install all dependencies with the line above

sorry, my bad :( it's fixed now